src/admin-events/admin-events.spec.js

```diff
@@ -0,0 +1,132 @@
+const ethers = require('ethers');
+const { getAddress } = require('@uma/contracts-node');
+
+// Load Config files
+const {
+  TransactionEvent,
+  FindingType,
+  FindingSeverity,
+  Finding,
+} = require('forta-agent');
+
+const config = require('../../agent-config.json');
```

Comment:
Same comment from the handler module:
```suggestion
const { umaEverestId } = require('../../agent-config.json');
```

Reply:
Done